types/iyzipay/index.d.ts

```diff
@@ -0,0 +1,1601 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+/**
+ * author: Can Gökçeaslan
+ * description: This peace of code is a typescript definition file for iyzipay nodejs sdk.
+ * website: https://www.cangokceaslan.com
+ * github: https://www.github.com/cangokceaslan
+ * npm: https://www.npmjs.com/package/@types/iyzipay
+ */
```

Comment:
Please consider sticking to explicit DT ownership, you would get discoverability by being owner of the types (published to NPM), giving linking back to your GH page.

Reply:
I have removed author details and other comments relating to me.